mosip/on-prem/README.md

```diff
@@ -44,9 +44,9 @@ ansible-playbook -i hosts.ini ports.yaml
 ansible-playbook -i hosts.ini swap.yaml
 ```
 ## Environment Check
```

Comment:
Environment Check and setup
1. Performs check to make sure localhost mapping is present in all nodes.
2. Adding these crons:....